modules/genomic_browser/php/cnv_browser.class.inc

```diff
@@ -140,15 +140,16 @@ class CNV_Browser extends \NDB_Menu_Filter
         $user = \User::singleton();
         if (!$user->hasPermission('genomic_browser_view_allsites')) {
             $site_arr     = implode(",", $user->getCenterIDs());
-            $this->query .= " AND candidate.CenterID IN (" . $site_arr . ")";
+            $this->query .= " AND candidate.RegistrationCenterID IN ("
```

Comment:
i think this should be the session centerID?

Reply:
or maybe not? im not too sure now, i don't know how genomic browser works

Reply:
I don't think there is any sessionID to attach genomic data. @xlecours could you confirm?